### PR TITLE
Add Time to celebrate blog post

### PR DESCRIPTION
**Pull request by olblak**

### CODE REVIEW

File: content/en/blog/2023/10/time_to_celebrate.adoc [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+title: "Time to celebrate"
```
Comment (lemeurherve) [inline content/en/blog/2023/10/time_to_celebrate.adoc:2]:
```suggestion
title: "Time to celebrate!"
```

File: content/en/blog/2023/10/time_to_celebrate.adoc [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+date: 2023-09-02T06:00:00+01:00
+draft: false
+weight: 50
+images: [""]
+contributors: ["olblak"]
+---
+
+= Time to celebrate
```
Comment (lemeurherve) [inline content/en/blog/2023/10/time_to_celebrate.adoc:10]:
```suggestion
= Time to celebrate!
```

File: content/en/blog/2023/10/time_to_celebrate.adoc [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+Foresight Mining is currently busy building the next generation of IOT collaborative CAD systems.
+
+image::/images/sponsors/foresightmining.jpeg["Foresight Mining",align="center"]
+
+== So what's coming next?
+
+As always it's hard to tell, but I guess finding the right balance between implementing crazy ideas, having fun, and fixing bugs we encounter as Updatecli usage grows.
+
+With a more concrete example, the biggest recent addition to Updatecli is the Terraform support contributed by Matthew Warman where using the declarative approach here combining the plugins **link:https://www.updatecli.io/docs/plugins/resource/terraform/file/[terraform/file]**, **link:https://www.updatecli.io/docs/plugins/resource/terraform/lock/[terraform/lock]**, **link:https://www.updatecli.io/docs/plugins/resource/terraform/provider/[terraform/provider]**, or **link:https://www.updatecli.io/docs/plugins/resource/terraform/registry/[terraform/registry]** (combined with any existing plugins), we can define tailored update pipeline for Terraform related project, or OpenTofu.
```
Comment (lemeurherve) [inline content/en/blog/2023/10/time_to_celebrate.adoc:49]:
```suggestion
With a more concrete example, the biggest recent addition to Updatecli is the Terraform support contributed by https://warman.io/[Matthew Warman] where using the declarative approach here combining the plugins **link:https://www.updatecli.io/docs/plugins/resource/terraform/file/[terraform/file]**, **link:https://www.updatecli.io/docs/plugins/resource/terraform/lock/[terraform/lock]**, **link:https://www.updatecli.io/docs/plugins/resource/terraform/provider/[terraform/provider]**, or **link:https://www.updatecli.io/docs/plugins/resource/terraform/registry/[terraform/registry]** (combined with any existing plugins), we can define tailored update pipeline for Terraform related project, or OpenTofu.
```

File: content/en/blog/2023/10/time_to_celebrate.adoc [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+In version 0.62.0, Matthew also contributed the Terraform Autodiscovery plugin that allows to handle automatically Terraform project updates.
```
Comment (lemeurherve) [inline content/en/blog/2023/10/time_to_celebrate.adoc:51]:
```suggestion
In version 0.62.0, Matthew also contributed the Terraform Autodiscovery plugin allowing to automatically handle Terraform project updates.
```

File: content/en/blog/2023/10/time_to_celebrate.adoc [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+For example, we can run commands such as `updatecli diff --experimental` to see what would be changed in Terraform project containing a Terraform lock file.
```
Comment (lemeurherve) [inline content/en/blog/2023/10/time_to_celebrate.adoc:52]:
```suggestion
For example, we can run commands such as `updatecli diff --experimental` to see what would change in Terraform project containing a Terraform lock file.
```

File: content/en/blog/2023/10/time_to_celebrate.adoc [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+On my side, I am exploring the possibility of using any OCI registries such as DockerHub or ghcr.io to share and reuse Updatecli policies.
```
Comment (lemeurherve) [inline content/en/blog/2023/10/time_to_celebrate.adoc:54]:
```suggestion
On my side, I am exploring the possibility of using OCI registries such as DockerHub or ghcr.io to share and reuse Updatecli policies.
```

**Review by lemeurherve — COMMENTED** [review]
Some small suggestions

File: content/en/blog/2023/10/time_to_celebrate.adoc (hunk 4 above)
Comment (lemeurherve) [reply content/en/blog/2023/10/time_to_celebrate.adoc:51]: I would also add a link on the terraform autodiscovery plugin (like the ones above for the miscellaneous TF plugins)

File: content/en/blog/2023/10/time_to_celebrate.adoc (hunk 2 above)
Comment (olblak) [reply content/en/blog/2023/10/time_to_celebrate.adoc:10]: I wonder why I can commit but I can't add suggestion to batch as you teach me

File: content/en/blog/2023/10/time_to_celebrate.adoc [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+
+Foresight Mining is currently busy building the next generation of IOT collaborative CAD systems.
+
+image::/images/sponsors/foresightmining.jpeg["Foresight Mining",align="center"]
+
+== So what's coming next?
+
+As always it's hard to tell, but I guess finding the right balance between implementing crazy ideas, having fun, and fixing bugs we encounter as Updatecli usage grows.
+
+With a more concrete example, the biggest recent addition to Updatecli is the Terraform support contributed by https://warman.io/[Matthew Warman] where using the declarative approach here combining the plugins **link:https://www.updatecli.io/docs/plugins/resource/terraform/file/[terraform/file]**, **link:https://www.updatecli.io/docs/plugins/resource/terraform/lock/[terraform/lock]**, **link:https://www.updatecli.io/docs/plugins/resource/terraform/provider/[terraform/provider]**, or **link:https://www.updatecli.io/docs/plugins/resource/terraform/registry/[terraform/registry]** (combined with any existing plugins), we can define tailored update pipeline for Terraform related project, or OpenTofu.
```
Comment (olblak) [inline content/en/blog/2023/10/time_to_celebrate.adoc:49]:
```suggestion
With a more concrete example, the biggest recent addition to Updatecli is the Terraform support contributed by **link:https://warman.io/[Matthew Warman]** where using the declarative approach here combining the plugins **link:https://www.updatecli.io/docs/plugins/resource/terraform/file/[terraform/file]**, **link:https://www.updatecli.io/docs/plugins/resource/terraform/lock/[terraform/lock]**, **link:https://www.updatecli.io/docs/plugins/resource/terraform/provider/[terraform/provider]**, or **link:https://www.updatecli.io/docs/plugins/resource/terraform/registry/[terraform/registry]** (combined with any existing plugins), we can define tailored update pipeline for Terraform related project, or OpenTofu.
```

Adding link and make it bold to better see it in the page

File: content/en/blog/2023/10/time_to_celebrate.adoc (hunk 4 above)
Comment (olblak) [reply content/en/blog/2023/10/time_to_celebrate.adoc:51]:
```suggestion
In version 0.62.0, Matthew also contributed the **link:https://www.updatecli.io/docs/plugins/autodiscovery/terraform/[Terraform Autodiscovery plugin]** that allows to handle automatically Terraform project updates.
```

File: content/en/blog/2023/10/time_to_celebrate.adoc [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+
+image::/images/sponsors/foresightmining.jpeg["Foresight Mining",align="center"]
+
+== So what's coming next?
+
+As always it's hard to tell, but I guess finding the right balance between implementing crazy ideas, having fun, and fixing bugs we encounter as Updatecli usage grows.
+
+With a more concrete example, the biggest recent addition to Updatecli is the Terraform support contributed by **link:https://warman.io/[Matthew Warman]** where using the declarative approach here combining the plugins **link:https://www.updatecli.io/docs/plugins/resource/terraform/file/[terraform/file]**, **link:https://www.updatecli.io/docs/plugins/resource/terraform/lock/[terraform/lock]**, **link:https://www.updatecli.io/docs/plugins/resource/terraform/provider/[terraform/provider]**, or **link:https://www.updatecli.io/docs/plugins/resource/terraform/registry/[terraform/registry]** (combined with any existing plugins), we can define tailored update pipeline for Terraform related project, or OpenTofu.
+
+In version 0.62.0, Matthew also contributed the Terraform Autodiscovery plugin allowing to automatically handle Terraform project updates.
```
Comment (olblak) [inline content/en/blog/2023/10/time_to_celebrate.adoc:51]:
```suggestion
In version 0.62.0, Matthew also contributed the **link:https://www.updatecli.io/docs/plugins/autodiscovery/terraform/[Terraform Autodiscovery plugin]** allowing to automatically handle Terraform project updates.
```

File: content/en/blog/2023/10/time_to_celebrate.adoc [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+
+== Let's Connect
+
+As usual, if you have any questions feel free to reach out via our **link:https://matrix.to/#/#Updatecli_community:gitter.im[chat]**
+
+And do not forget to star our project on GitHub repository **link:https://github.com/updatecli/updatecli/stargazers[updatecli/updatecli]** as it's probably one of the easiest ways to show your appreciation.
+
+== The People ❤️
+
+The Updatecli wouldn't stand here today with all the people who contributed to it.
```
Comment (olblak) [inline content/en/blog/2023/10/time_to_celebrate.adoc:65]:
```suggestion
The Updatecli wouldn't stand here today without all the people who contributed to it.
```

File: content/en/blog/2023/10/time_to_celebrate.adoc [hunk 10]
```diff
@@ -0,0 +1,69 @@
...
+
+Foresight Mining is currently busy building the next generation of IOT collaborative CAD systems.
+
+image::/images/sponsors/foresightmining.jpeg["Foresight Mining",align="center"]
+
+== So what's coming next?
+
+As always it's hard to tell, but I guess finding the right balance between implementing crazy ideas, having fun, and fixing bugs we encounter as Updatecli usage grows.
+
+With a more concrete example, the biggest recent addition to Updatecli is the Terraform support contributed by **link:https://warman.io/[Matthew Warman]** where using the declarative approach here combining the plugins **link:https://www.updatecli.io/docs/plugins/resource/terraform/file/[terraform/file]**, **link:https://www.updatecli.io/docs/plugins/resource/terraform/lock/[terraform/lock]**, **link:https://www.updatecli.io/docs/plugins/resource/terraform/provider/[terraform/provider]**, or **link:https://www.updatecli.io/docs/plugins/resource/terraform/registry/[terraform/registry]** (combined with any existing plugins), we can define tailored update pipeline for Terraform related project, or OpenTofu.
```
Comment (olblak) [inline content/en/blog/2023/10/time_to_celebrate.adoc:49]:
```suggestion
With a more concrete example, the biggest recent addition to Updatecli is the Terraform support contributed by **link:https://warman.io/[Matthew Warman]** where using the declarative approach combining plugins **link:https://www.updatecli.io/docs/plugins/resource/terraform/file/[terraform/file]**, **link:https://www.updatecli.io/docs/plugins/resource/terraform/lock/[terraform/lock]**, **link:https://www.updatecli.io/docs/plugins/resource/terraform/provider/[terraform/provider]**, or **link:https://www.updatecli.io/docs/plugins/resource/terraform/registry/[terraform/registry]** (or any others existing plugins), we can define tailored update pipeline for Terraform related project, or OpenTofu.
```

File: content/en/blog/2023/10/time_to_celebrate.adoc (hunk 2 above)
Comment (lemeurherve) [reply content/en/blog/2023/10/time_to_celebrate.adoc]: @olblak you can add suggestions to batch only on the "files" tab, not on the "conversation" one.

Comment (olblak) [reply content/en/blog/2023/10/time_to_celebrate.adoc]: haaaa that explain, why I managed to  batch some suggestions :) 
Thank you

File: content/en/blog/2023/10/time_to_celebrate.adoc [hunk 11]
```diff
@@ -0,0 +1,69 @@
+---
+title: "Time to celebrate!"
+date: 2023-09-02T06:00:00+01:00
```
Comment (olblak) [inline content/en/blog/2023/10/time_to_celebrate.adoc:3]:
```suggestion
date: 2023-10-02T12:00:00+01:00
```

**Pull request merged**